Foodee/ember-foxy-forms
# Update dependencies for ember-auto-loader 2

# Wat it do? 

* This PR is a dependency update, so no user-facing functionality has changed.
* Several libraries have been updated to support `ember-auto-loader@^2` [requirement](https://github.com/ef4/ember-auto-import/blob/main/docs/upgrade-guide-2.0.md) for Ember addons that use that it as a dependency, which Foxy Forms does.
* The biggest change to the library code is the replacement of `ember-popper` with `ember-tooltips`, a more modern alternative with, for our purposes, the same API. 
* If we want to retain the `ember-popper` dependency (as it is used throughout the Masterfox codebase) that is definitely possible, but will require more time to upgrade, as it's quite old and I would have to update quite a bit of stuff.
* Switches package manager from NPM to Yarn.
* Adds [Volta](https://volta.sh/), which helps teams ensure all members are using the same node, and yarn version.
* The github actions `ci.yml` file has been updated to lock the node/yarn versions to those described in the `volta:` section of `package.json`.

# How to test?

* Visit the following url to load the documentation page: (WIP)
* Ensure all form inputs are working as expected.
